glean-core/src/common_metric_data.rs

```diff
@@ -73,9 +75,10 @@ pub struct CommonMetricData {
     pub dynamic_label: Option<String>,
 }
 
-#[derive(Default, Debug)]
+#[derive(Default, Debug, MallocSizeOf)]
 pub struct CommonMetricDataInternal {
     pub inner: CommonMetricData,
+    #[ignore_malloc_size_of = "atomic integers never allocate"]
```

Comment:
Is this a bug in `wr_malloc_size_of` that it doesn't already know and handle this?

Reply:
Well. "Bug". Just not implemented. It could be done (and I have the keys to do it!). But for now just skipping it works just as good.

Reply:
Please file and reference a bug (I'm guessing from the moz.build it'd be in `Core:: Graphics: WebRender`?)